testsuite/integration-tests-spring/pom.xml

```diff
@@ -20,7 +20,7 @@
     </properties>
 
     <modules>
-        <module>deployment</module>
+<!--        <module>deployment</module>-->
```

Comment:
Does this not work? I ran it locally and the tests passed for me.

Reply:
oops. I missed it :p